datasets/clinc_oos/dataset_infos.json

```diff
@@ -0,0 +1 @@
", "order", "jump_start", "schedule_meeting", "meeting_schedule", "freeze_account", "what_song", "meaning_of_life", "restaurant_reservation", "traffic", "make_call", "text", "bill_balance", "improve_credit_score", "change_language", "no", "measurement_conversion", "timer", "flip_coin", "do_you_have_pets", "balance", "tell_joke", "last_maintenance", "exchange_rate", "uber", "car_rental", "credit_limit", "oos", "shopping_list", "expiration_date", "routing", "meal_suggestion", "tire_change", "todo_list", "card_declined", "rewards_balance", "change_accent", "vaccines", "reminder_update", "food_last", "change_ai_name", "bill_due", "who_do_you_work_for", "share_location", "international_visa", "calendar", "translate", "carry_on", "book_flight", "insurance_change", "todo_list_update", "timezone", "cancel_reservation", "transactions", "credit_score", "report_fraud", "spending_history", "directions", "spelling", "insurance", "what_is_your_name", "reminder", "where_are_you_from", "distance", "payday", "flight_status", "find_phone", "greeting", "alarm", "order_status", "confirm_reservation", "cook_time", "damaged_card", "reset_settings", "pin_change", "replacement_card_duration", "new_card", "roll_dice", "income", "taxes", "date", "who_made_you", "pto_request", "tire_pressure", "how_old_are_you", "rollover_401k", "pto_request_status", "how_busy", "application_status", "recipe", "calendar_update", "play_music", "yes", "direct_deposit", "credit_limit_change", "gas", "pay_bill", "ingredients_list", "lost_luggage", "goodbye", "what_can_i_ask_you", "book_hotel", "are_you_a_bot", "next_song", "change_speed", "plug_type", "maybe", "w2", "oil_change_when", "thank_you", "shopping_list_update", "pto_balance", "order_checks", "travel_alert", "fun_fact", "sync_device", "schedule_maintenance", "apr", "transfer", "ingredient_substitution", "calories", "current_location", "international_fees", "calculator", "definition", "next_holiday", "update_playlist", "mpg", "min_payment", "change_user_name", "restaurant_suggestion", "travel_notification", "cancel", "pto_used", "travel_suggestion", "change_volume"], "names_file": null, "id": null, "_type": "ClassLabel"}}, "post_processed": null, "supervised_keys": null, "builder_name": "clinc150", "config_name": "plus", "version": {"version_str": "1.0.0", "description": "", "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 791255, "num_examples": 15250, "dataset_name": "clinc150"}, "validation": {"name": "validation", "num_bytes": 160302, "num_examples": 3100, "dataset_name": "clinc150"}, "test": {"name": "test", "num_bytes": 286970, "num_examples": 5500, "dataset_name": "clinc150"}}, "download_checksums": {"https://raw.githubusercontent.com/clinc/oos-eval/master/data/data_oos_plus.json": {"num_bytes": 2509789, "checksum": "bfcca9ae515623541dc1983c94c4ed7cae9d26b42ae47d74b972e51bb6f7a21f"}}, "download_size": 2509789, "post_processing_size": null, "dataset_size": 1238527, "size_in_bytes": 3748316}}
```

Comment:
```suggestion
{"small": {"description": "    This dataset is for evaluating the performance of intent classification systems in the\n    presence of \"out-of-scope\" queries. By \"out-of-scope\", we mean queries that do not fall\n    into any of the system-supported intent classes. Most datasets include only data that is\n    \"in-scope\". Our dataset includes both in-scope and out-of-scope data. You might also know\n    the term \"out-of-scope\" by other terms, including \"out-of-domain\" or \"out-of-distribution\".\n\nSmall, in which there are only 50 training queries per each in-scope intent\n", "citation": "    @inproceedings{larson-etal-2019-evaluation,\n    title = \"An Evaluation Dataset for Intent Classification and Out-of-Scope Prediction\",\n    author = \"Larson, Stefan  and\n      Mahendran, Anish  and\n      Peper, Joseph J.  and\n      Clarke, Christopher  and\n      Lee, Andrew  and\n      Hill, Parker  and\n      Kummerfeld, Jonathan K.  and\n      Leach, Kevin  and\n      Laurenzano, Michael A.  and\n      Tang, Lingjia  and\n      Mars, Jason\",\n    booktitle = \"Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP)\",\n    year = \"2019\",\n    url = \"https://www.aclweb.org/anthology/D19-1131\"\n}\n", "homepage": "https://github.com/clinc/oos-eval/", "license": "", "features": {"text": {"dtype": "string", "id": null, "_type": "Value"}, "intent": {"num_classes": 151, "names": ["restaurant_reviews", "nutrition_info", "account_blocked", "oil_change_how", "time", "weather", "redeem_rewards", "interest_rate", "gas_type", "accept_reservations", "smart_home", "user_name", "report_lost_card", "repeat", "whisper_mode", "what_are_your_hobbies", "order", "jump_start", "schedule_meeting", "meeting_schedule", "freeze_account", "what_song", "meaning_of_life", "restaurant_reservation", "traffic", "make_call", "text", "bill_balance", "improve_credit_score", "change_language", "no", "measurement_conversion", "timer", "flip_coin", "do_you_have_pets", "balance", "tell_joke", "last_maintenance", "exchange_rate", "uber", "car_rental", "credit_limit", "oos", "shopping_list", "expiration_date", "routing", "meal_suggestion", "tire_change", "todo_list", "card_declined", "rewards_balance", "change_accent", "vaccines", "reminder_update", "food_last", "change_ai_name", "bill_due", "who_do_you_work_for", "share_location", "international_visa", "calendar", "translate", "carry_on", "book_flight", "insurance_change", "todo_list_update", "timezone", "cancel_reservation", "transactions", "credit_score", "report_fraud", "spending_history", "directions", "spelling", "insurance", "what_is_your_name", "reminder", "where_are_you_from", "distance", "payday", "flight_status", "find_phone", "greeting", "alarm", "order_status", "confirm_reservation", "cook_time", "damaged_card", "reset_settings", "pin_change", "replacement_card_duration", "new_card", "roll_dice", "income", "taxes", "date", "who_made_you", "pto_request", "tire_pressure", "how_old_are_you", "rollover_401k", "pto_request_status", "how_busy", "application_status", "recipe", "calendar_update", "play_music", "yes", "direct_deposit", "credit_limit_change", "gas", "pay_bill", "ingredients_list", "lost_luggage", "goodbye", "what_can_i_ask_you", "book_hotel", "are_you_a_bot", "next_song", "change_speed", "plug_type", "maybe", "w2", "oil_change_when", "thank_you", "shopping_list_update", "pto_balance", "order_checks", "travel_alert", "fun_fact", "sync_device", "schedule_maintenance", "apr", "transfer", "ingredient_substitution", "calories", "current_location", "international_fees", "calculator", "definition", "next_holiday", "update_playlist", "mpg", "min_payment", "change_user_name", "restaurant_suggestion", "travel_notification", "cancel", "pto_used", "travel_suggestion", "change_volume"], "names_file": null, "id": null, "_type": "ClassLabel"}}, "post_processed": null, "supervised_keys": null, "builder_name": "clinc_oos", "config_name": "small", "version": {"version_str": "1.0.0", "description": "", "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 394128, "num_examples": 7600, "dataset_name": "clinc_oos"}, "validation": {"name": "validation", "num_bytes": 160302, "num_examples": 3100, "dataset_name": "clinc_oos"}, "test": {"name": "test", "num_bytes": 286970, "num_examples": 5500, "dataset_name": "clinc_oos"}}, "download_checksums": {"https://raw.githubusercontent.com/clinc/oos-eval/master/data/data_small.json": {"num_bytes": 1702451, "checksum": "050e17476e6b4fa88f8518edaf09921c8f5e3a86dc8b63615361102a20b2ac01"}}, "download_size": 1702451, "post_processing_size": null, "dataset_size": 841400, "size_in_bytes": 2543851}, "imbalanced": {"description": "    This dataset is for evaluating the performance of intent classification systems in the\n    presence of \"out-of-scope\" queries. By \"out-of-scope\", we mean queries that do not fall\n    into any of the system-supported intent classes. Most datasets include only data that is\n    \"in-scope\". Our dataset includes both in-scope and out-of-scope data. You might also know\n    the term \"out-of-scope\" by other terms, including \"out-of-domain\" or \"out-of-distribution\".\n\nImbalanced, in which intents have either 25, 50, 75, or 100 training queries.\n", "citation": "    @inproceedings{larson-etal-2019-evaluation,\n    title = \"An Evaluation Dataset for Intent Classification and Out-of-Scope Prediction\",\n    author = \"Larson, Stefan  and\n      Mahendran, Anish  and\n      Peper, Joseph J.  and\n      Clarke, Christopher  and\n      Lee, Andrew  and\n      Hill, Parker  and\n      Kummerfeld, Jonathan K.  and\n      Leach, Kevin  and\n      Laurenzano, Michael A.  and\n      Tang, Lingjia  and\n      Mars, Jason\",\n    booktitle = \"Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP)\",\n    year = \"2019\",\n    url = \"https://www.aclweb.org/anthology/D19-1131\"\n}\n", "homepage": "https://github.com/clinc/oos-eval/", "license": "", "features": {"text": {"dtype": "string", "id": null, "_type": "Value"}, "intent": {"num_classes": 151, "names": ["restaurant_reviews", "nutrition_info", "account_blocked", "oil_change_how", "time", "weather", "redeem_rewards", "interest_rate", "gas_type", "accept_reservations", "smart_home", "user_name", "report_lost_card", "repeat", "whisper_mode", "what_are_your_hobbies", "order", "jump_start", "schedule_meeting", "meeting_schedule", "freeze_account", "what_song", "meaning_of_life", "restaurant_reservation", "traffic", "make_call", "text", "bill_balance", "improve_credit_score", "change_language", "no", "measurement_conversion", "timer", "flip_coin", "do_you_have_pets", "balance", "tell_joke", "last_maintenance", "exchange_rate", "uber", "car_rental", "credit_limit", "oos", "shopping_list", "expiration_date", "routing", "meal_suggestion", "tire_change", "todo_list", "card_declined", "rewards_balance", "change_accent", "vaccines", "reminder_update", "food_last", "change_ai_name", "bill_due", "who_do_you_work_for", "share_location", "international_visa", "calendar", "translate", "carry_on", "book_flight", "insurance_change", "todo_list_update", "timezone", "cancel_reservation", "transactions", "credit_score", "report_fraud", "spending_history", "directions", "spelling", "insurance", "what_is_your_name", "reminder", "where_are_you_from", "distance", "payday", "flight_status", "find_phone", "greeting", "alarm", "order_status", "confirm_reservation", "cook_time", "damaged_card", "reset_settings", "pin_change", "replacement_card_duration", "new_card", "roll_dice", "income", "taxes", "date", "who_made_you", "pto_request", "tire_pressure", "how_old_are_you", "rollover_401k", "pto_request_status", "how_busy", "application_status", "recipe", "calendar_update", "play_music", "yes", "direct_deposit", "credit_limit_change", "gas", "pay_bill", "ingredients_list", "lost_luggage", "goodbye", "what_can_i_ask_you", "book_hotel", "are_you_a_bot", "next_song", "change_speed", "plug_type", "maybe", "w2", "oil_change_when", "thank_you", "shopping_list_update", "pto_balance", "order_checks", "travel_alert", "fun_fact", "sync_device", "schedule_maintenance", "apr", "transfer", "ingredient_substitution", "calories", "current_location", "international_fees", "calculator", "definition", "next_holiday", "update_playlist", "mpg", "min_payment", "change_user_name", "restaurant_suggestion", "travel_notification", "cancel", "pto_used", "travel_suggestion", "change_volume"], "names_file": null, "id": null, "_type": "ClassLabel"}}, "post_processed": null, "supervised_keys": null, "builder_name": "clinc_oos", "config_name": "imbalanced", "version": {"version_str": "1.0.0", "description": "", "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 546909, "num_examples": 10625, "dataset_name": "clinc_oos"}, "validation": {"name": "validation", "num_bytes": 160302, "num_examples": 3100, "dataset_name": "clinc_oos"}, "test": {"name": "test", "num_bytes": 286970, "num_examples": 5500, "dataset_name": "clinc_oos"}}, "download_checksums": {"https://raw.githubusercontent.com/clinc/oos-eval/master/data/data_imbalanced.json": {"num_bytes": 2016773, "checksum": "4886730b20c51eece26aa392ecae2717bfe9908680419e96255351c6148eb4cc"}}, "download_size": 2016773, "post_processing_size": null, "dataset_size": 994181, "size_in_bytes": 3010954}, "plus": {"description": "    This dataset is for evaluating the performance of intent classification systems in the\n    presence of \"out-of-scope\" queries. By \"out-of-scope\", we mean queries that do not fall\n    into any of the system-supported intent classes. Most datasets include only data that is\n    \"in-scope\". Our dataset includes both in-scope and out-of-scope data. You might also know\n    the term \"out-of-scope\" by other terms, including \"out-of-domain\" or \"out-of-distribution\".\n\nOOS+, in which there are 250 out-of-scope training examples, rather than 100.\n", "citation": "    @inproceedings{larson-etal-2019-evaluation,\n    title = \"An Evaluation Dataset for Intent Classification and Out-of-Scope Prediction\",\n    author = \"Larson, Stefan  and\n      Mahendran, Anish  and\n      Peper, Joseph J.  and\n      Clarke, Christopher  and\n      Lee, Andrew  and\n      Hill, Parker  and\n      Kummerfeld, Jonathan K.  and\n      Leach, Kevin  and\n      Laurenzano, Michael A.  and\n      Tang, Lingjia  and\n      Mars, Jason\",\n    booktitle = \"Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP)\",\n    year = \"2019\",\n    url = \"https://www.aclweb.org/anthology/D19-1131\"\n}\n", "homepage": "https://github.com/clinc/oos-eval/", "license": "", "features": {"text": {"dtype": "string", "id": null, "_type": "Value"}, "intent": {"num_classes": 151, "names": ["restaurant_reviews", "nutrition_info", "account_blocked", "oil_change_how", "time", "weather", "redeem_rewards", "interest_rate", "gas_type", "accept_reservations", "smart_home", "user_name", "report_lost_card", "repeat", "whisper_mode", "what_are_your_hobbies", "order", "jump_start", "schedule_meeting", "meeting_schedule", "freeze_account", "what_song", "meaning_of_life", "restaurant_reservation", "traffic", "make_call", "text", "bill_balance", "improve_credit_score", "change_language", "no", "measurement_conversion", "timer", "flip_coin", "do_you_have_pets", "balance", "tell_joke", "last_maintenance", "exchange_rate", "uber", "car_rental", "credit_limit", "oos", "shopping_list", "expiration_date", "routing", "meal_suggestion", "tire_change", "todo_list", "card_declined", "rewards_balance", "change_accent", "vaccines", "reminder_update", "food_last", "change_ai_name", "bill_due", "who_do_you_work_for", "share_location", "international_visa", "calendar", "translate", "carry_on", "book_flight", "insurance_change", "todo_list_update", "timezone", "cancel_reservation", "transactions", "credit_score", "report_fraud", "spending_history", "directions", "spelling", "insurance", "what_is_your_name", "reminder", "where_are_you_from", "distance", "payday", "flight_status", "find_phone", "greeting", "alarm", "order_status", "confirm_reservation", "cook_time", "damaged_card", "reset_settings", "pin_change", "replacement_card_duration", "new_card", "roll_dice", "income", "taxes", "date", "who_made_you", "pto_request", "tire_pressure", "how_old_are_you", "rollover_401k", "pto_request_status", "how_busy", "application_status", "recipe", "calendar_update", "play_music", "yes", "direct_deposit", "credit_limit_change", "gas", "pay_bill", "ingredients_list", "lost_luggage", "goodbye", "what_can_i_ask_you", "book_hotel", "are_you_a_bot", "next_song", "change_speed", "plug_type", "maybe", "w2", "oil_change_when", "thank_you", "shopping_list_update", "pto_balance", "order_checks", "travel_alert", "fun_fact", "sync_device", "schedule_maintenance", "apr", "transfer", "ingredient_substitution", "calories", "current_location", "international_fees", "calculator", "definition", "next_holiday", "update_playlist", "mpg", "min_payment", "change_user_name", "restaurant_suggestion", "travel_notification", "cancel", "pto_used", "travel_suggestion", "change_volume"], "names_file": null, "id": null, "_type": "ClassLabel"}}, "post_processed": null, "supervised_keys": null, "builder_name": "clinc_oos", "config_name": "plus", "version": {"version_str": "1.0.0", "description": "", "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 791255, "num_examples": 15250, "dataset_name": "clinc_oos"}, "validation": {"name": "validation", "num_bytes": 160302, "num_examples": 3100, "dataset_name": "clinc_oos"}, "test": {"name": "test", "num_bytes": 286970, "num_examples": 5500, "dataset_name": "clinc_oos"}}, "download_checksums": {"https://raw.githubusercontent.com/clinc/oos-eval/master/data/data_oos_plus.json": {"num_bytes": 2509789, "checksum": "bfcca9ae515623541dc1983c94c4ed7cae9d26b42ae47d74b972e51bb6f7a21f"}}, "download_size": 2509789, "post_processing_size": null, "dataset_size": 1238527, "size_in_bytes": 3748316}}
```
`clinc150` -> `clinc_oos`